ruby_programming/testing_with_rspec/test_driven_development.md

```diff
@@ -40,3 +40,12 @@ As usual, it depends. Still, here are some reasons we think it might be importan
 2. For a more rounded take, consider DHH's ["TDD is Dead. Long Live Testing"](http://david.heinemeierhansson.com/2014/tdd-is-dead-long-live-testing.html), which sparked controversial conversations on what TDD is, what it has become, and what it should be.
 
 </div>
+
+### Knowledge Check
+
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+- <a class="knowledge-check-link" href="https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-programming/lessons/test-driven-development#what-is-test-driven-development">What is the "red-green-refactor" cycle?</a>
```

Comment:
```suggestion
- <a class="knowledge-check-link" href="#what-is-test-driven-development">What is the "red-green-refactor" cycle?</a>
```

Reply:
I had not realized there were two suggested changes ..
I fixed it!